src/highlighting.ts

```diff
@@ -107,7 +109,7 @@ export class Highlighter implements vscode.DocumentSemanticTokensProvider {
       '--format',
       'json',
       filePath,
-      '--stdin',
+      // '--stdin',
```

Comment:
This is because for some reason this prevents `highlight` command from working.
I am investigating this, but commenting this out allows everything work properly

Reply:
I fixed this in https://github.com/anoma/juvix/pull/2043
So this will work with that changes in place 🙂